subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -11,6 +11,7 @@
 #include <sys/slist.h>
 #include <sys/util.h>
 
+#ifndef ULL_LLCP_UNITTEST
```

Comment:
This can be replaced with the more generic ZTEST_UNITTEST instead of ULL_LLCP_UNITTEST

Reply:
Yes will do that instead